manifests/server.pp

```diff
@@ -297,6 +298,23 @@
     }
   }
 
+  if $automatic_dlo_cleanup {
+    if $facts['systemd'] {
+      # deploy a systemd timer + service to cleanup old reports
+      # https://puppet.com/docs/puppetdb/5.2/maintain_and_tune.html#clean-up-the-dead-letter-office
+      systemd::unit_file{'puppetdb-cleanup.service':
```

Comment:
We should rename this to `puppetdb-dlo-cleanup.service` (and the matching timer one below) in case we want to add another cleanup service in the future.

Reply:
sounds good, done